client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -37,6 +37,17 @@ const resultsSpeak = debounceSpeak( { message: 'Search results loaded.' } );
 
 const errorSpeak = debounceSpeak( { message: 'No search results found.' } );
 
+const filterManagePurchaseLink = ( hasPurchases, section ) => ( { post_id } ) => {
+	if (
+		post_id === 111349 &&
+		! hasPurchases &&
+		! [ 'purchases', 'site-purchases' ].includes( section )
+	) {
+		return false;
+	}
+	return true;
+};
```

Comment:
Could also be:
```js
function filterManagePurchaseLink( hasPurchases, isPurchasesSection ) {
  if ( hasPurchases || isPurchasesSection ) {
    return () => true;
  }
  return ( { post_id } ) => post_id !== 111349;
}
```

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/55978/commits/aede20d1a2171ddbb0a7c1c1b2180fc5f2b4bb86